lib/src/signals/logic.dart

```diff
@@ -319,13 +319,59 @@ class Logic {
   void operator <=(Logic other) => gets(other);
 
   /// Logical bitwise NOT.
-  Logic operator ~() => NotGate(this).out;
+  Logic operator ~() {
+    if (this is Const) {
+      if (value.isValid) {
```

Comment:
I think we can rely on the operators within `LogicValue` for this?  i.e. something like `Const(~this.value)`?